park_info.clj

```diff
@@ -0,0 +1,367 @@
+        ;---Directions---
+        :public_transport "stop: Klamovka - buses #149, #191, #217, - trams #4, #7, #9, #10, #58, #59"
+        :GPS_coords "50°4'17,04" N, 14°22'39,72" E"
+        :parking false
+        :parking_extra nil
+        ;---Additional Information---
+        :dogs true
+        :path_type "paved & asphalt"
+        :administered_by "Municipality Prague 5"
+        :opening_hours "Open year-round"
+        ;---Map---
+        :map_url "https://www.praha.eu/public/16/84/48/184296_4_klamovka_mapa.pdf"       
+       
+    }
+    def Ladronka{
+        ;---Basic Info---
+        :food true
+        :food_extra "restaurant Ladronka (www.ladronka.com)"
+        :toilet true
+        :toilet_extra "in the restaurant"
+        :POIs "Large area suited for free time activities and relaxation for all age categories, year-long: musical performances and social events, suited for a day-long stay"
+        ;---Children and Sport---
+        :bike_path true
+        :skating true
+        :sports_field false
+        :sports_field_extra: "Not in the park itself, but nearby: traffic playfround and Vypich Youth House"
+        :playground true
+        :playground_extra "by the restaurant and by the Ladronka bus stop"
+        ;---Directions---
+        :public_transport "
+        stop: U ladronky/ Rozýnova/ Štefkova - bus #191
+        stop: Vypich - buses #108, #174, #180, #191, #I3, - trams: #15, 22, 25, 57"
+        :GPS_coords "50°4'47.447" N, 14°21'52.445" E"
+        :parking true
+        :parking_extra "Yes, in streets Tomanova & Skokanská"
+        ;---Additional Information---
+        :dogs true
+        :path_type "paved, threshing & asphalt"
+        :administered_by "Municipality Prague 6"
+        :opening_hours "Open year-round"
+        ;---Map---
+        :map_url "https://www.praha.eu/public/72/f3/b/184218_4_ladronka_mapa.pdf"       
+       
+    }
+    def Letna{
+        ;---Basic Info---
+        :food true
+        :food_extra "Letenský estate (www.letenskyzamecek.cz), Hanavský pavilion (www.hanavskypavilon.cz) "
+        :toilet true
+        :toilet_extra "in the restaurants"
+        :POIs "Spacious area for free time activities, suitable for all age categories, year-long: cultural events, suited for a day-long stay"
+        ;---Children and Sport---
+        :bike_path true
+        :skating true
+        :sports_field false
+        :sports_field_extra: "Not in the park itself, however sports and recreational activities are permitted on grassy segments, additionally:
+        2x skates rentals: Rental #1 - Skala sport tel. +420 605 258 670, Rental #2 - tel. +420 220 571 721
+        General purpose rental: AC Sparta rental - tel. +420 603 938 328,
+        small pitch for football, paid access tennis courts on premises of LTC Slavia - tel. +420 777 789 140"
+        :playground true
+        :playground_extra "4 in total. By the letenský estate, 2 by the tennis courts, one by Hanavský pavillion"
```

Comment:
Include number of playgrounds in each location or not at all.

Reply:
Is a textual description provided by the park, copied verbatim from [source](https://www.praha.eu/jnp/cz/co_delat_v_praze/parky/letna/index.html) (Děti a sport tab). Personally do not see this as an issue.